### PR TITLE
improve runtime connect error messages

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
Improves the error message for following commands if the runtime is not started / reachable:
- ocis list
- ocis kill
- ocis run

The error message now looks like this: `Failed to connect to the runtime. Is the runtime running and did you configure the right runtime address ("localhost:9250")`

## Related Issue

- Fixes [Fix error message when the runtime has not started but you file a command for it](https://github.com/owncloud/ocis/issues/3458)

## Motivation and Context
Make oCIS commands more human friendly

## How Has This Been Tested?

- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by mmattel** [comment]
Great, thanks a lot.
Adding myself here to get notified when the change gets merged 😄

File: ocis/pkg/command/kill.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func KillCommand(cfg *config.Config) *cli.Command {
 		Action: func(c *cli.Context) error {
 			client, err := rpc.DialHTTP("tcp", net.JoinHostPort(cfg.Runtime.Host, cfg.Runtime.Port))
 			if err != nil {
-				log.Fatal("dialing:", err)
+				log.Fatalf("Failed to connect to the runtime. Is the runtime running and did you configure the right runtime address (\"%s\")", cfg.Runtime.Host+":"+cfg.Runtime.Port)
```
Comment (mmattel) [inline ocis/pkg/command/kill.go:38]:
```suggestion
				log.Fatalf("Failed to connect to the runtime. Has the runtime been started and did you configure the right runtime address (\"%s\")", cfg.Runtime.Host+":"+cfg.Runtime.Port)
```

File: ocis/pkg/command/run.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ func RunCommand(cfg *config.Config) *cli.Command {
 		Action: func(c *cli.Context) error {
 			client, err := rpc.DialHTTP("tcp", net.JoinHostPort(cfg.Runtime.Host, cfg.Runtime.Port))
 			if err != nil {
-				log.Fatal("dialing:", err)
+				log.Fatalf("Failed to connect to the runtime. Is the runtime running and did you configure the right runtime address (\"%s\")", cfg.Runtime.Host+":"+cfg.Runtime.Port)
```
Comment (mmattel) [inline ocis/pkg/command/run.go:39]:
```suggestion
				log.Fatalf("Failed to connect to the runtime. Has the runtime been started and did you configure the right runtime address (\"%s\")", cfg.Runtime.Host+":"+cfg.Runtime.Port)
```

**Comment by mmattel** [comment]
I have added a small text improvement. If you agree, just add to batch (in files changed tab) and commit directly from the browser.

**Review by micbar — APPROVED** [review]
LGTM

**Pull request merged**